### ISSUE TITLE
Using without CUDA GPUs

### ISSUE THREAD

**Issue by banterle** [ssundaram21/dreamsim#18]
Would it be possible to have in the next release the possibility to run the metric on non-CUDA GPUs, please? 
This would be helpful for some machines in which CUDA GPUs are not possible to have.

**Comment by imneonizer** [comment]
I would also like to see if we can run this on ios, `.mlcode` format, as existing CLIP models can be converted to mlcode.

**Comment by ssundaram21** [comment]
Hi --this should be already possible for any device supported by PyTorch (we have tested on MPS).

To load the model on CPU:
`model, preprocess = dreamsim(pretrained=True, device="cpu")`

To load the model on a non-CUDA GPU (such as MPS):
`model, preprocess = dreamsim(pretrained=True, device="mps")`

Feel free to re-open this issue if the above gives any problems!
